### DISCUSSION TITLE
Calyx Fest 2023

### DISCUSSION THREAD

**Discussion by rachitnigam** [cucapra/futil#1811] [General]
Starting a top-level thread to ask questions and track ideas that came up during Calyx fest 2023!

**Comment by rachitnigam** [comment]
**PIFO Trees** by @anshumanmohan

**Reply by sampsyo** [reply]
This is a long discussion, but: how hard would it be to build the scaffolding required to connect Calyx-generated designs to the Ethernet ports on a Xilinx SmartNIC? Our Alveo U50, for example, can connect to the network...
